app/controllers/programs_controller.rb

```diff
@@ -1,6 +1,4 @@
 class ProgramsController < ApplicationController
-  before_action :require_admin, only: [:new, :create, :edit, :update, :destroy]
```

Comment:
Can you fill me in on why you removed this or at least confirm that it was intentional before I merge?